packages/sonarwhal/docs/contributor-guide/getting-started/events.md

```diff
@@ -77,10 +78,23 @@ type FetchError {
 }
 ```
 
-## `fetch::start`
+## `fetch::missing::<resource-type>`
+
+Event is emitted **when** a resource is not found.
+
+**Format:**
+
+```ts
+type FetchMissing {
+    /** The URL to download */
+    resource: string;
```

Comment:
When the resource is missing, I thing that most of the time we don't now the url/path

Reply:
I forgot to change this to `/** The URL of the target. */`. 

Also:
  * `resource` this should be optional (for cases that do not have it)? 
  * Maybe this should also include an `element` also optional for cases such as `<link rel="manifest" href="">`? Or should that be another case?

Reply:
> `resource` this should be optional (for cases that do not have it)?

I think so.